x-pack/plugins/security_solution/server/usage/detections/detection_rule_helpers.ts

```diff
@@ -253,9 +293,9 @@ export const getDetectionRuleMetrics = async (
         const isElastic = isElasticRule(hit._source?.alert.tags);
         return {
           rule_name: hit._source?.alert.name,
-          rule_id: ruleId,
+          rule_id: hit._source?.alert.params.ruleId,
```

Comment:
NOTE: I changed this as well which is different from the original PR. This was changed in 8.0/8.1 already, but since I added e2e, I figured I should just fix it rather than changing the tests. It will make back porting anything else easier.